src/leiningen/new/chestnut/dev/user.clj

```diff
@@ -23,10 +35,11 @@
     (clojure.java.shell/sh "lein" "auto" "sassc" "once")))
 {{/sass?}}
 
-(def http-handler
-  (wrap-reload #'{{project-ns}}.server/http-handler))
+(set-refresh-dirs "src" "dev")
```

Comment:
There's something funny going with the whitespace/newlines in this file. I'm getting the impression Leiningen changed how it handles whitespace around conditional blocks, with the result that there's no newline between `(defn dev-system ...)` and `(set-refresh-dir)`. This seems to work

``` clojure
(defn dev-system []
  (merge
   ({{project-ns}}.server/prod-system)
   (component/system-map
    :figwheel-system (sys/figwheel-system (config/fetch-config))
    :css-watcher (sys/css-watcher {:watch-paths ["resources/public/css"]}))))
{{#less?}}
(defn start-less []
  (future
    (println "Starting less.")
    (clojure.java.shell/sh "lein" "less" "auto")))
{{/less?}}
{{#sass?}}
(defn start-sass []
  (future
    (println "Starting sass.")
    (clojure.java.shell/sh "lein" "auto" "sassc" "once")))
{{/sass?}}

(set-refresh-dirs "src" "dev")
(reloaded.repl/set-init! #(dev-system))
```